docs/data/charts/lines/lines.md

```diff
@@ -52,6 +52,16 @@ By default, they are stacked in the order you defined them, with positive values
 
 For more information, see [stacking docs](/x/react-charts/stacking/).
 
+## Missing data
```

Comment:
```suggestion
## Partial data
```

```suggestion
## Incomplete data
```

I feel like "missing" looks to much like it's abnormal.

Reply:
Yeah, both are great alternatives. I also felt the same, but couldn't  think of a better alternative at the time. 🙈